internal/stateindex/index_entries_test.go

```diff
@@ -422,6 +429,9 @@ func TestIndexEntriesOfExistingValues(t *testing.T) {
 		"age": types.Type_NUMBER,
 	}
 
+	encoded25 := encodeOrderPreservingVarUint64(uint64(25))
+	encoded26 := encodeOrderPreservingVarUint64(uint64(26))
```

Comment:
I would also check the corner cases of 0, +/- max-int, and verify that the ordering of a mix of positive and negative numbers is as expected.

Reply:
Partially Done. I have created issue #145 I can add the order testing only after https://github.com/IBM-Blockchain/bcdb-server/pull/131 gets merged. I would in fact add this test in the query executor side as well.